tests/bin/polykey.test.ts

```diff
@@ -1,11 +1,69 @@
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import readline from 'readline';
+import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
 import * as execUtils from '../utils/exec';
 import { runTestIfPlatforms } from '../utils';
 
 describe('polykey', () => {
-  runTestIfPlatforms('lunix', 'docker')('default help display', async () => {
+  runTestIfPlatforms('linux', 'docker')('default help display', async () => {
```

Comment:
@tegefaulkes why does this require a conditional check?

Reply:
That should've been removed. I must've missed it.